frontend/src/pages/register/formValidation.tsx

```diff
@@ -100,6 +102,15 @@ const FormValidation = () => {
     );
   };
 
+  const tos = (
+    <PlainText>
+      I accept {` `}
```

Comment:
"I accept {` `}"?

Reply:
it was necessary because prettier removes any unnecessary spaces, but we need it here to separate the text from the link.